### ISSUE TITLE
Question/Feature Add... jira list -a unassigned

### ISSUE THREAD

**Issue by criffel** [Netflix-Skunkworks/go-jira#162]
!!! Thank you for this tool. Makes my life that much easier at work. !!! 

JQL has unassigned equal to 'IS EMPTY' in query fields while trying to search for unassigned tickets... Has this already been thought of in your script? if so, how would I jira list -p project -a unassigned ?

Again, thank you!

**Comment by coryb** [comment]
Not sure I understand.  You can just run `jira ls --query "..."` to use jql directly.  Or you can create a file:
`$HOME/.jira.d/list.yml` with `query: ...` property to set the query as the default.

**Comment by criffel** [comment]
Let me rephrase: You already give us the option to run
jira list -p project -a user
 - which lists all tickets under project assigned to user
Did you include an option to search for all tickets under a project that are unassigned? If not I will just create a yml to do it but was wondering if you already had this option available...

**Comment by coryb** [comment]
I dont think that is supported currently, I believe using the `--query` option to provide your own JQL is the only want at the moment.

**Comment by criffel** [comment]
No worries. I thank you all for your work on this tool. Very useful!
